submit-web/src/components/Submission/SubmissionItemReviewConfirmation.tsx

```diff
@@ -85,11 +85,14 @@ export default function SubmissionItemReviewConfirmation({
             },
           });
         }}
+        onCancel={() => {
```

Comment:
and what if we wanted cancel to just cancel on do nothing? With this change cancel will always click onClick